hudi-integ-test/src/main/java/org/apache/hudi/integ/testsuite/HoodieDeltaStreamerWrapper.java

```diff
@@ -78,7 +78,7 @@ public JavaRDD<WriteStatus> compact() throws Exception {
   public Pair<SchemaProvider, Pair<String, JavaRDD<HoodieRecord>>> fetchSource() throws Exception {
     DeltaSync service = getDeltaSync();
     service.refreshTimeline();
-    return service.readFromSource(service.getCommitTimelineOpt());
+    return service.readFromSource(service.getCommitTimelineOpt(), service.getDeltaCommitTimelineOpt());
```

Comment:
getCommitsTimeline?

Reply:
The definition of the method **readFromSource** changed to:

public Pair<SchemaProvider, Pair<String, JavaRDD<HoodieRecord>>> readFromSource(Option<HoodieTimeline> commitTimelineOpt, Option<HoodieTimeline> deltaCommitTimelineOpt)

added **deltaCommitTimelineOpt** to the param list in order to resume from the correct checkpoint for mor table which just changed from cow table.

Reply:
I will cc @bvaradar for code reviewing.